.pre-commit-config.yaml

```diff
@@ -81,6 +81,33 @@ repos:
             ^python/cudf/cudf/core/dtypes.py|
             ^python/cudf/cudf/tests/pytest.ini
           )
+      - id: cython-docstrings
+        name: cython-docstrings
+        description: 'Enforce multi-line Cython docstrings start on line two'
+        # We want to allow """One line docstring"""
```

Comment:
This comment explaining the purpose is really long. Can we write this description elsewhere? Could this be a hook we invoke from a Python file, and/or centralized in https://github.com/rapidsai/pre-commit-hooks?

Reply:
This is doable, if we think it is definitively worth it.